MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/service/database/SentimentsDatabase.java

```diff
@@ -0,0 +1,43 @@
+package com.google.moviestvsentiments.service.database;
+
+import android.content.Context;
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
+import com.google.moviestvsentiments.model.Account;
+import com.google.moviestvsentiments.service.account.AccountDao;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * SentimentsDatabase provides a high-level interface for performing database queries.
+ */
+@Database(entities = {Account.class}, version = 1, exportSchema = false)
+public abstract class SentimentsDatabase extends RoomDatabase {
+
+    /**
+     * accountDao returns an object that allows queries on the accounts table.
```

Comment:
nit: You can consider using "@return". (also for other files in this commit) (https://drive.google.com/file/d/14mmUPsrAP1EVyHmrNom_NLbMhhzklw2t/view?usp=sharing)

Reply:
I didn't use @return in this specific instance because https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-block-tags states block tags "never appear with an empty description." Using @return and complying with the style guide would require a description and an @return clause that say basically the same thing.

Reply:
Make sense.